### PR TITLE
Get-DbaPermission.Tests - reorder the integration tests and add debug messages

### LINKED ISSUE #7101: [Bug] Get-DbaPermission tests failure
The test is unstable and continues to fail in Appveyor runs due to recent changes that were made to it.

@lancasteradam can you look at this section of the test that you added? The context block is failing most commonly on test runs in Appveyor with PRs now:
```
Describe : Get-DbaPermission Integration Tests
Context  : Ensure implicit permissions are included in the result set
Name     : It the dbo user and db_owner users are returned in the result set with the CONTROL permission
Result   : Failed
Message  : Expected 2, but got $null.
```

https://github.com/sqlcollaborative/dbatools/blob/2329ce5720adb7c59e73ff232804b3d8cfaa348d/tests/Get-DbaPermission.Tests.ps1#L112-L126

### PR DESCRIPTION
**Pull request by lancasteradam**
Get-DbaPermission.Tests - reorder the integration tests and add debug messages

## Type of Change

 - [x] Bug fix (non-breaking change, fixes #7101 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [x] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [x] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [x] Documentation
 - [ ] Build system
 

### Purpose
 
Debugging the integration test in appveyor. The tests are working fine on my local machine with SQL 2019, 2017, and 2016. 

### Approach
There is one test that uses a mock method for resolving the network name, so I've put that test last in the file in case there is some unexpected side effect from that test.

### CODE REVIEW

File: tests/Get-DbaPermission.Tests.ps1 [hunk 1]
```diff
@@ -66,7 +66,9 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
         $table2 = New-DbaDbTable -SqlInstance $server -Database $dbName -Name $tableName2 -Schema $schemaNameForTable2 -ColumnMap $tableSpec2
 
         # debugging errors seen only in AppVeyor
-        Write-Message -Level Verbose -Message "Get-DbaPermission: Server=$server, dbName=$dbName, loginDBO=$($loginDBO.Name), loginDBOwner=$($loginDBOwner.Name), loginUser1=$($loginUser1.Name), newUserDBOwner=$($newUserDBOwner.Name), newUser1=$($newUser1.Name), table1=$($table1.Name), loginUser2=$($loginUser2.Name), newUser2=$($newUser2.Name), table2=$($table2.Name), table2Schema=$($table2.Schema)"
+        Write-Message -Level Warning -Message "Get-DbaPermission: Server=$server, dbName=$dbName, loginDBO=$($loginDBO.Name), loginDBOwner=$($loginDBOwner.Name), loginUser1=$($loginUser1.Name), newUserDBOwner=$($newUserDBOwner.Name), newUser1=$($newUser1.Name), table1=$($table1.Name), loginUser2=$($loginUser2.Name), newUser2=$($newUser2.Name), table2=$($table2.Name), table2Schema=$($table2.Schema)"
```
Comment (wsmelton) [inline tests/Get-DbaPermission.Tests.ps1:69]: Write-Message does not need to be used, please use `Write-Host` instead.

Comment (lancasteradam) [reply tests/Get-DbaPermission.Tests.ps1:69]: ok, will do

**Comment by lancasteradam** [comment]
@wsmelton, the "Mock Resolve-DbaNetworkName { $null }" seems like a possible reason why the test was failing in a VM environment. I've removed that integration test because it doesn't seem related to the command. So far the Get-DbaPermission tests have run successfully three times in a row in the appveyor env.

**Comment by wsmelton** [comment]
We'll hope for the best. The likelihood is on the creation of objects because we are using so many other commands in these tests they don't formally error when they can't create the object, which ends up being why tests fail because of `$null`.

I'll address all of that as I rework the tests for Pester 5+.

**Review by wsmelton — APPROVED** [review]

**Pull request merged**